examples/configuration/README.md

```diff
@@ -0,0 +1,66 @@
+# Example - Get Configuration
+
+This example demonstrates the Configuration APIs in Dapr.
+It demonstrates the following APIs:
+- **configuration**: Get configuration from statestore
+
+> **Note:** Make sure to use the latest proto bindings
+
+## Pre-requisites
+
+- [Dapr CLI and initialized environment](https://docs.dapr.io/getting-started)
+- [Install Python 3.7+](https://www.python.org/downloads/)
+
+## Install Dapr python-SDK
+
+<!-- Our CI/CD pipeline automatically installs the correct version, so we can skip this step in the automation -->
+```bash
+pip3 install dapr dapr-ext-grpc
+```
+
+## Store the configuration in configurationstore 
+<!-- STEP
+name: Set configuration value
+expected_stdout_lines:
+  - "OK"
+timeout_seconds: 5
+-->
+
+```bash
+docker exec dapr_redis redis-cli SET greeting "hello world||1"
+```
+
+<!-- END_STEP -->
+
+## Run the example
+
+Change directory to this folder:
+```bash
+cd examples/configuration
+```
+
+To run this example, use the following command:
+
+<!-- STEP
+name: Run get configuration example
+expected_stdout_lines:
+  - "== APP == Got key=greeting value=hello world version=1"
+timeout_seconds: 5
```

Comment:
5 seconds might be pushing it a little bit. We have on several occasions had known issues where daprd startup is delayed by more than 5s on certain platforms.